### ISSUE TITLE
[core] unified search chips are not in tab sequence

### ISSUE THREAD

**Issue by susnux** [nextcloud/server#58385]
The chips for filters added are not in the correct tab sequence.

<img width="530" height="239" alt="Image" src="https://github.com/user-attachments/assets/6aa64532-c077-457a-9857-df45e34f9b35" />

They should be directly accessible in tab sequence after the filters itself.

**Comment by susnux** [comment]
https://github.com/nextcloud/server/pull/63224
